nlp/environment.yml

```diff
@@ -1,9 +1,18 @@
 name: birdy
+
 channels:
-  - conda-forge
-  - defaults
+- defaults
+- conda-forge
+
 dependencies:
-  - intake-esm
-  - intake-stac
-  - sat-search
-  - threddsclient
```

Comment:
I don't think these four packages should have been removed from this file. These were the initial dependencies required by the nlp team, and should still be found in this file.

Reply:
@ahandan Sorry for misleading you on this one, I didn't know these dependencies were useful in this image.. !